karma/makeconf.js

```diff
@@ -68,6 +69,10 @@ function makeConf(defaultFile, browserNoActivityTimeout, requires) {
             '--js-flags="--expose-gc"'
           ]
         },
+        ElectronWebRTC: {
+          base: 'Electron',
+          flags: ['--show', '--no-custom-user-agent'],
```

Comment:
@makarandp0 Please remove `--show` so that we can enable this for Travis in a future release. Alternatively, you can accept an environment variable `SHOW` as follows:
```js
const electronFlags = ['--no-custom-user-agent'];
if ('SHOW' in process.env) {
  electronFlags.push('--show');
}
...
ElectronWebRTC: {
  ...
  flags: electronFlags
  ...
}
...
```

Reply:
okay, i will remove show for now.